### PR TITLE
Tooltip: global options

### LINKED ISSUE #5686: Tooltip delay
Add an option to delay the showing and hiding of tooltips.

https://atomiks.github.io/tippyjs/v6/customization/

### PR DESCRIPTION
**Pull request by stsrki**
Closes #5686

Added an option to define global tooltip settings.

### CODE REVIEW

File: Source/Blazorise/BlazoriseOptions.cs [hunk 1]
```diff
@@ -124,5 +124,23 @@ public string LicenseKey
...
+    /// </summary>
+    public BlazoriseTooltipOptions TooltipOptions { get; set; }
+
     #endregion
+}
+
+/// <summary>
+/// Represents the options available for configuring the behavior of a Tooltip in Blazorise.
+/// </summary>
+public class BlazoriseTooltipOptions
```
Comment (David-Moreira) [inline Source/Blazorise/BlazoriseOptions.cs:138]: I was more of a separate file for every class in the past and I believe I've seen similar reviews from you in the past.
Personally, nowadays I don't mind keeping related stuff in the same file. What about you? Are you fine with keeping both of these in the same file?

File: Source/Blazorise/Components/Tooltip/Tooltip.razor.cs [hunk 2]
```diff
@@ -305,7 +314,7 @@ public TooltipTrigger Trigger
     /// <summary>
     /// Specifies the delay in ms once a trigger event is fired before a Tooltip shows or hides.
     /// </summary>
-    [Parameter] public (int Show, int Hide) Delay { get; set; } = (0, 0);
+    [Parameter] public (int Show, int Hide)? Delay { get; set; }
```
Comment (David-Moreira) [inline Source/Blazorise/Components/Tooltip/Tooltip.razor.cs:317]: What about `ShowDelay` and `HideDelay` as two separate parameters? 
I still don't like how we changed our mind about using values tuples, I would almost certainly say we don't have any other api with value tuples. I like them more for internal stuff and less for public stuff.
I'd just keep it consistent and use the same as we've been doing.

I honestly think the two separate params are more clear as a public api. I don't mind either anyway.

**Review by David-Moreira — APPROVED** [review]

File: Source/Blazorise/Components/Tooltip/Tooltip.razor.cs (hunk 2 above)
Comment (stsrki) [reply Source/Blazorise/Components/Tooltip/Tooltip.razor.cs:317]: The more I'm thinking, you're right. It is better not to use a tuple and to have a separate parameters instead. Going to change it now.

File: Source/Blazorise/BlazoriseOptions.cs (hunk 1 above)
Comment (stsrki) [reply Source/Blazorise/BlazoriseOptions.cs]: Done

**Review by David-Moreira — APPROVED** [review]

**Pull request merged**